Plutarch/Internal/Lift.hs

```diff
@@ -271,11 +271,8 @@ of a @Data@ encoding).
 newtype DeriveBuiltinPLiftable (a :: S -> Type) (h :: Type) (s :: S)
   = DeriveBuiltinPLiftable (a s)
   deriving stock (Generic)
-  deriving anyclass (PlutusType)
-
--- | @since WIP
-instance DerivePlutusType (DeriveBuiltinPLiftable a h) where
-  type DPTStrat _ = PlutusTypeNewtype
+  deriving anyclass (SOP.Generic)
+  deriving (PlutusType) via (DeriveFakePlutusType (DeriveBuiltinPLiftable a h))
```

Comment:
`@since` got lost here
```suggestion
  deriving anyclass (SOP.Generic)
  deriving (
  -- | @since wip
  PlutusType
  ) via (DeriveFakePlutusType (DeriveBuiltinPLiftable a h))
```